File: Web/ClientApp/src/app/models/db/languageLang.ts
import { EntityBase } from "../entityBase";

export class LanguageLang extends EntityBase {
    languageCode: string;
    languageCodeForname: string;
    languageName: string;

    constructor(languageCode?: string, languageCodeForname?: string) {
        super()
        if (languageCode) this.languageCode = languageCode
        if (languageCodeForname) this.languageCodeForname = languageCodeForname
    }
}

File: Domain/Entities/DB/LanguageLang.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities.DB;

public class LanguageLang: EntityBase
{
    [MaxLength(20)]
    public string LanguageCode { get; set; }
    [MaxLength(20)]
    public string LanguageCodeForname { get; set; }
    [MaxLength(200)]
    public string LanguageName { get; set; }
}
